zp-relayer/services/price-feed/OneInchPriceFeed.ts

```diff
@@ -29,29 +27,27 @@ export class OneInchPriceFeed implements IPriceFeed {
     this.contract = new web3.eth.Contract(OracleAbi as AbiItem[], contractAddress)
   }
 
+  async init() {
```

Comment:
Could you clarify when this code is being invoked? I tried to run the relayer with the dynamic fee manager and got the following error:
```
dev-zkbob-relayer  | /app/node_modules/number-to-bn/node_modules/bn.js/lib/bn.js:2329
dev-zkbob-relayer  |     assert(!num.isZero());
dev-zkbob-relayer  |                 ^
dev-zkbob-relayer  | 
dev-zkbob-relayer  | TypeError: Cannot read properties of undefined (reading 'isZero')
dev-zkbob-relayer  |     at BN.divmod (/app/node_modules/number-to-bn/node_modules/bn.js/lib/bn.js:2329:17)
dev-zkbob-relayer  |     at BN.div (/app/node_modules/number-to-bn/node_modules/bn.js/lib/bn.js:2425:17)
dev-zkbob-relayer  |     at OneInchPriceFeed.convert (/app/zp-relayer/services/price-feed/OneInchPriceFeed.js:42:42)
dev-zkbob-relayer  |     at /app/zp-relayer/services/fee/FeeManager.js:46:34
dev-zkbob-relayer  |     at DynamicFeeOptions.mapI (/app/zp-relayer/services/fee/FeeManager.js:24:28)
dev-zkbob-relayer  |     at DynamicFeeOptions.convert (/app/zp-relayer/services/fee/FeeManager.js:46:14)
dev-zkbob-relayer  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
dev-zkbob-relayer  |     at async DynamicFeeManager.fetchFeeOptions (/app/zp-relayer/services/fee/FeeManager.js:142:31)
dev-zkbob-relayer  |     at async /app/zp-relayer/services/fee/FeeManager.js:123:32
dev-zkbob-relayer  |     at async setIntervalAndRun (/app/zp-relayer/utils/helpers.js:100:5)
dev-zkbob-relayer  |     at async DynamicFeeManager.start (/app/zp-relayer/services/fee/FeeManager.js:122:41)
dev-zkbob-relayer  |     at async init (/app/zp-relayer/init.js:92:5)
```

Reply:
Sorry, forgot to test this after recent changes. Now, it should be ok